lib/plugins/track.js

```diff
@@ -64,7 +64,9 @@ function configureValues (track) {
           : document.location.href.split('#')[0]
       },
       td_user_agent: function () {
-        return window.navigator.userAgent
+        var userAgent = window.navigator.userAgent
+
+        return [userAgent, 'JSSDK/' + version].join(';')
```

Comment:
Nitpicking:
Either declare both `userAgent` and `sdkUserAgent`, or inline `window.navigator.userAgent` to this line.